src/pubsub/core/callables.py

```diff
@@ -27,6 +27,11 @@
 # consistent with the MDS of the topic to which listener is being subscribed.
 UserListener = Callable[..., Any]
 
+class NO_DEFAULT:
+    """A placeholder for an arg or kwarg without a default."""
+    def __repr__(self):
+        return "NO_DEFAULT"
+
```

Comment:
Can this class perhaps just be replaced with something like:
`NO_DEFAULT = "NO_DEFAULT"`

Why does it need to be a new class?

Reply:
I used a class, because it is going to be unique no matter what the user has an input for their function that says "NO_DEFAULT"?

For example, say my function was:
```
def test(myText):
    print(myText)
```
And I wanted to send: `test("NO_DEFAULT")`. That would not work, because that value is being used. So, a class special class allows the user to have any inputs they want.